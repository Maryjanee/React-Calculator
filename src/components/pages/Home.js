/* eslint-disable max-len */
const Home = () => (
  <div className="container">
    <h1>Welcome to Our Page</h1>
    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Qui accusantium earum voluptatum expedita dolor voluptate consequuntur sed saepe perferendis. Reiciendis ratione quod corporis quasi molestiae magni, vel obcaecati voluptas hic?</p>
    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Qui accusantium earum voluptatum expedita dolor voluptate consequuntur sed saepe perferendis. Reiciendis ratione quod corporis quasi molestiae magni, vel obcaecati voluptas hic?</p>
  </div>
);

export default Home;
