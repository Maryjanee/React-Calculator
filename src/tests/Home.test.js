import TestRenderer from 'react-test-renderer';
import Home from '../components/pages/Home';

it('Snapshot test for Home component', () => {
    const homeShot = TestRenderer
    .create(<Home />)
    .toJSON();
  expect(homeShot).toMatchSnapshot();
}) 