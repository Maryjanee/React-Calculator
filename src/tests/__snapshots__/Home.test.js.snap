// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for Home component 1`] = `
<div
  className="container"
>
  <h1>
    Welcome to Our Page
  </h1>
  <p>
    Warm greetings from the creators of the Math Magicians Application
  </p>
  <p>
    We are thrilled to have you stop by an do some work here. Thank you for your time.
  </p>
</div>
`;
